integrations/Fullstory/camelcase.js

```diff
@@ -0,0 +1,51 @@
+// util function to convert the input to string type
```

Comment:
I wouldn't put this in FullStory as it is a generic utility.

Reply:
got it